src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side1;
+
+    public Square(String color, int side1) {
+        super(color);
+        this.side1 = side1;
+    }
+
+    public int getArea() {
+        return side1 * side1;
+    }
```

Comment:
The method 'getArea' should be named to reflect that it's calculating the area, such as 'calculateArea'. Moreover, it should be part of an interface as the task checklist suggests separating behavior into interfaces if the methods are conceptually different.